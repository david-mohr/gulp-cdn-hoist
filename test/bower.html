<html>
<head>
  <!-- build:js /vendor.js -->
  <script src="bower_components/jquery/dist/jquery.js"></script>
  <script src="bower_components/angular/angular.js"></script>
  <script src="local.js"></script>
  <!-- endbuild -->
  <!-- build:js /app.js -->
  <script src="local.js"></script>
  <!-- endbuild -->
</head>
</html>
