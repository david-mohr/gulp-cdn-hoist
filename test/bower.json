{
    "name": "testing only",
    "dependencies": {
        "jquery": "~2.1.4",
        "angular": "~1.4.7"
    }
}
