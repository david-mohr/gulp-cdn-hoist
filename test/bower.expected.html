<html>
<head>
  <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
<script src="//ajax.googleapis.com/ajax/libs/angularjs/1.4.7/angular.min.js"></script>
<script src="/vendor.js"></script>
  
<script src="/app.js"></script>
</head>
</html>
